src/bundle/Resources/public/scss/_grid-view-item.scss

```
@@ -9,6 +9,9 @@
     background: $ibexa-color-white;
     box-shadow: calculateRem(4px) calculateRem(2px) calculateRem(17px) 0 rgba($ibexa-color-black, 0.05);
     transition: all $ibexa-admin-transition-duration $ibexa-admin-transition;
+    min-width: calculateRem(216px);
+    max-width: calculateRem(216px);
+    margin: 0 calculateRem(14px) calculateRem(31.5px);
```

Discussed in private: probably there is no need to use 0.5px
```suggestion
    margin: 0 calculateRem(14px) calculateRem(32px);
```